src/tape/TapePlayer.cpp

```diff
@@ -30,20 +31,21 @@ TapePlayer::TapePlayer(const char* path)
 	playbackState = std::make_unique<PlaybackState>(pathBin.c_str());
 
 	yamlRoot = YAML::LoadFile(pathYaml);
-	yamlRecording = yamlRoot["recording"];
 
-	if (yamlRoot[RGL_VERSION]["major"].as<int>() != RGL_VERSION_MAJOR ||
-	    yamlRoot[RGL_VERSION]["minor"].as<int>() != RGL_VERSION_MINOR) {
-		throw RecordError("recording version does not match rgl version");
+	auto version = getTapeCall(findFirst({RGL_VERSION}).value()).getArgsNode();
```

Comment:
I believe it may fail (pre-refactor tape format version or broken tape). Please add handling for it.

Reply:
Yes, it should be sufficient to pull out `findFirst({RGL_VERSION})` and check if it `.has_value()`. If not, then report an error.